lib/language_pack/helpers/nodebin.rb

```diff
@@ -0,0 +1,24 @@
+require 'json'
+
+class LanguagePack::Nodebin
+  URL = "https://nodebin.herokai.com/v1/"
+
+  def self.query(q)
+    response = Net::HTTP.get_response(URI("#{URL}/#{q}"))
+    if response.code == '200'
+      JSON.parse(response.body)
+    end
+  end
+
+  def self.node(q)
+    query("node/linux-x64/#{q}")
+  end
+
+  def self.node_lts
+    node("latest?range=6.x")
```

Comment:
I'm down with the idea & if @jmorrell wants to implement 👍  but I wouldn't sweat the maintenance burden too much (once a year updates).